enterprise/install-scripts/openshift.ks

```diff
@@ -1248,7 +1258,7 @@ YUM
 configure_cart_repos()
 { # add cartridge (or dependency) repo as needed
   local -A url=(
-          [jbosseap]="${jboss_repo_base}/jbeap/6/os"
+          [jbosseap]="${jboss_repo_base}/jbeap/${jbosseap_version:-6}/os"
```

Comment:
Oh, *now* I see that you're using `$jbosseap_version` to derive both the path component here and the channel name below.  Might be more straightforward for `set_defaults()` just to set `$jbosseap_role` and `$jbosseap_version` or similar to simplify and consolidate the logic.  (I would suggest `$jbosseap_repo_path` and `$jbosseap_rhn_channel` rather than `$jbosseap_version`, but I guess we luck out in that the substring in the RHN channel name happens to be the same substring as the path component in the Yum repository name.)